### PR TITLE
Changes for i18n

### PR DESCRIPTION
**Pull request by ODOWDAIBM**
Signed-off-by: Anthony O'Dowd <a_o-dowd@uk.ibm.com>

#### Type of change

- Documentation update

#### Description

Concepts, task and reference material for international language translators using new i18n repository.

#### Additional details

None

#### Related issues

To support new i18n repo and myriad of new language translations being created

#### Release Note
Into Master, and retro to 2.2 would be great.

### CODE REVIEW

**Comment by ODOWDAIBM** [comment]
For reviewers:  New topics are:

- https://hyperledger-fabric--1569.org.readthedocs.build/en/1569/CONTRIBUTING.html#contributing-documentation
- https://hyperledger-fabric--1569.org.readthedocs.build/en/1569/advice_for_writers.html
- https://hyperledger-fabric--1569.org.readthedocs.build/en/1569/docs_guide.html
- https://hyperledger-fabric--1569.org.readthedocs.build/en/1569/international_languages.html

File: docs/source/advice_for_writers.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+workgroups](https://wiki.hyperledger.org/display/fabric/International+groups).
+See what your favorite workgroup is doing, and get connected with them.
+Each workgroup has a list of members and their contact information.
+
+## Other ways to connect
+
+Hyperledger Fabric has many other collaboration mechanisms such as mailing
+lists, contributor meetings and maintainer meetings. Find out about these and
+more [here](./contributing.html).
+
```
Comment (pamandrejko) [inline docs/source/advice_for_writers.md:71]: Link did not work. 
assuming it should just be 
Find out about these and
more [here](contributing.html).

File: docs/source/docs_guide.md [hunk 2]
```diff
@@ -181,8 +263,34 @@ instructions show you how to use ReadTheDocs to do this.
...
+Use `fabric-docs-i18n` instead of `fabric` in the above instructions if you're
+building an international language translation.
+
+Now, anytime you modify or add documentation content to your fork, this URL will
+automatically get updated with your changes!
+
+## Making a PR
+
+You can submit your PR for inclusion using the following
+[instructions](./github/github.html#opening-a-pull-request-in-github).
```
Comment (pamandrejko) [inline docs/source/docs_guide.md:275]: So many people are submitting doc updates without signing their requests. I think we have to drive them down this path instead https://hyperledger-fabric.readthedocs.io/en/latest/github/github.html
where it shows them to do the commit -s

File: docs/source/international_languages.md [hunk 3]
```diff
@@ -1,275 +1,230 @@
...
+Before your new language can be published to the documentation website, you must
+translate the following topics.  These topics help users and translators of your
+new language get started.
+
+* [Fabric front page](https://hyperledger-fabric.readthedocs.io/zh_CN/latest/)
+
+  This is your advert! Thanks to you, users can now see that the documentation
+  is available in their language. It might not be complete yet, but its clear
+  you and your team are trying to achieve. Translating tis page will help you
+  recruit other translators.
```
Comment (pamandrejko) [inline docs/source/international_languages.md:154]: typo tis

**Review by pamandrejko — APPROVED** [review]
This is absolutely fantastic Anthony for both English and non-English content. Thank you for putting all of this together. 


There were a few nits that I can address after this is merged.

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.1

**Comment by pamandrejko** [comment]
@Mergifyio backport release-2.2

**Pull request merged**